src/main/java/com/scmp/framework/testng/listeners/SuiteListener.java

```diff
@@ -180,4 +186,71 @@ private void createNewTestRun(ISuite suite, String projectId, String testRunName
 			frameworkLogger.warn("No Test Run created as no test cases were found.");
 		}
 	}
+
+	/**
+	 * Log consecutive failed test cases on master branch
```

Comment:
@benliscmp can we add some details on how this function works? say base on what configs and how it identify consecutive failures?

Reply:
Added comment